src/utils/metakg/parser/index.py

```diff
@@ -0,0 +1,80 @@
+from .endpoint import Endpoint
```

Comment:
the name `index.py` is from the node version, does not make sense in Python. I would put it into `__init__.py`